patches/chrome-browser-ui-toolbar-media_router_contextual_menu.cc.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/chrome/browser/ui/toolbar/media_router_contextual_menu.cc b/chrome/browser/ui/toolbar/media_router_contextual_menu.cc
+index 46b5dd8c2b246bb830ce3ae38e977d94d051d64e..29c300491aeb6c5ff4f5adb181bd77954e6339c4 100644
+--- a/chrome/browser/ui/toolbar/media_router_contextual_menu.cc
++++ b/chrome/browser/ui/toolbar/media_router_contextual_menu.cc
+@@ -135,12 +135,18 @@ bool MediaRouterContextualMenu::IsCommandIdVisible(int command_id) const {
+ 
+ void MediaRouterContextualMenu::ExecuteCommand(int command_id,
+                                                int event_flags) {
++#if defined(BRAVE_CHROMIUM_BUILD)
++  const char kAboutPageUrl[] = "https://community.brave.com/";
```

Comment:
we have a lot of these urls that just go to a generic community page and that doesn't seem very user friendly. Maybe we should add paths for these to bring people to more relevant info?

Reply:
agree but if we block on waiting for web pages to be available we won't make a lot of progress.

Reply:
Am making a list of such links and will have specific community topics created but would need to know if they go under specific brave-core section or in general help section

cc: @kjozwiak